scripts/buildUtils.mjs

```diff
@@ -273,11 +275,21 @@ export function createBuildConfigs() {
           moduleIds: 'named',
         },
         output: {
+          devtoolModuleFilenameTemplate(info) {
```

Comment:
this is to get relative paths in the sourcemaps instead of the absolute path from the machine of the person running the build